src/fishbmc_addon.h

```diff
@@ -39,13 +39,13 @@ class ATTR_DLL_LOCAL CVisualizationFishBMC : public kodi::addon::CAddonBase,
   CVisualizationFishBMC();
   ~CVisualizationFishBMC() override;
 
-  bool Start(int channels, int samplesPerSec, int bitsPerSample, std::string songName) override;
+  bool Start(int channels, int samplesPerSec, int bitsPerSample, std::string songName);
```

Comment:
```suggestion
  bool Start(int channels, int samplesPerSec, int bitsPerSample, const std::string& songName) override;
```

ref: https://github.com/xbmc/xbmc/blob/9fc8af5d9cce4cc96c3ca7471049cc538f56b80f/xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Visualization.h#L472

Note that this will require changes to fishbmc_addon.cpp also.